client/app/services/user.js

```diff
@@ -33,18 +34,14 @@ function disableUser(user) {
   return $http
     .post(disableResource(user))
     .then((data) => {
-      toastr.warning(`User <b>${userName}</b> is now disabled.`, { allowHtml: true });
+      notification.warning(`User ${userName} is now disabled.`);
```

Comment:
I dunno if this should be a warning.. maybe `notification.info`?

Reply:
@ranbena in this case it's okay - we show message that user was disabled, and admin should see that they probably did something unsafe

Reply:
I thought it annoying with toastr, but with notification it turns out to be okay (that's why I've opted not to change it during the replacement)

IMO both options are suitable here, but if you still think it's better with `info` I can simply change that @ranbena  :)

Reply:
Let's leave as is.